src/pages/saksbehandling/steg/beregning/DelerAvPeriodeInputs.tsx

```diff
@@ -0,0 +1,71 @@
+import { FormikErrors } from 'formik';
+import { Normaltekst, Feilmelding } from 'nav-frontend-typografi';
+import React from 'react';
+import DatePicker from 'react-datepicker';
+import { IntlShape } from 'react-intl';
+
+import { FradragFormData } from '~pages/saksbehandling/steg/beregning/FradragInputs';
+import { DelerAvPeriode } from '~types/Fradrag';
+
+import styles from './fradragInputs.module.less';
+
+const DelerAvPeriodeInputs = (props: {
+    fraOgMedId: string;
+    tilOgMedId: string;
+    fradrag: FradragFormData;
+    setFieldValue: (field: string, value: Date | [Date, Date] | null) => void;
+    delerAvPeriodeErrors: FormikErrors<DelerAvPeriode> | undefined;
+    intl: IntlShape;
+}) => {
+    const fraOgMedError = props.delerAvPeriodeErrors?.fraOgMed;
+    const tilOgMedError = props.delerAvPeriodeErrors?.tilOgMed;
+
+    return (
+        <div className={styles.delerAvPeriodeContainer}>
+            <div>
+                <Normaltekst className={styles.dateLabelTekst}>
+                    {props.intl.formatMessage({ id: 'datovelger.fom.legend' })}
+                </Normaltekst>
+                <DatePicker
+                    id={props.fraOgMedId}
+                    name={props.fraOgMedId}
+                    selected={props.fradrag.delerAvPeriode?.fraOgMed}
+                    onChange={(e) => {
+                        props.setFieldValue(props.fraOgMedId, e);
+                    }}
+                    dateFormat="MM/yyyy"
+                    showMonthYearPicker
+                    isClearable
+                    selectsEnd
+                    startDate={props.fradrag.delerAvPeriode?.fraOgMed}
+                    endDate={props.fradrag.delerAvPeriode?.tilOgMed}
+                    minDate={props.fradrag.delerAvPeriode?.fraOgMed}
```

Comment:
Blir disse riktige når det er input kun for `fraOgMed`?

Reply:
Tror ikke jeg helt forstå hva du mente. De er velt ment bare for å begrense hvilke datoer man har muligheten til å velge.

Reply:
Jeg tror det blir litt kluss. Scenario:
1. Bruker velger 15. september -- `startDate` og `minDate` vil nå være 15. september
2. Bruker har lyst til å endre til 12. september, men får ikke lov fordi startDate` og `minDate` er satt til 15. september

Eller har jeg misforstått?

Reply:
De skal vel teknisk bare velge måneder, ikke det? Og hvis så, så har man muligheten til å endre dem ved å krysse av på datepicker, og legge inn en ny dato. Men ja, det er slik den i praksis fungerer per nå

Reply:
Men vil ikke casen være den samme når det er snakk om å velge mnd? Altså at man ikke kan velge august etter å ha valgt september. Fiksen er jo i så fall bare å fjerne spesifisering av `startDate` og `minDate`-propsene.

Reply:
Jeg bare fjernet minDate propsen, slik at når du klikker på date pickeren, så kan du fortsatt velge en tidligere dato.
for tilOgMed, gjorde jeg at minDaten blir fraOgMed, slik at det ikke skjer at bruker trykker på en tidligere dato enn fraOgMed. 

Den beholder den kule date-rangen 👍